### ISSUE TITLE
[Question] Persistent notification

### ISSUE THREAD

**Issue by LuanTeles** [aldostools/webMAN-MOD#895]
Is there a way to make a pop up persistent, just like holding start+select to make the system info pop up persistent? and a command to release it?

**Comment by aldostools** [comment]
Try this:
/popup.ps3$	- show persistent info ON
/popup.ps3*	- show persistent info OFF

**Comment by LuanTeles** [comment]
It works but only displays the system information and make it persistent, if i use /popup.ps3$Test it will act like a normal notification, not persistent

can the command support custom text?

**Comment by aldostools** [comment]
The command works according to the documentation.

I have check what is needed to support custom text. If it is not too difficult I could add this feature.

**Comment by LuanTeles** [comment]
> 

Thanks in advance Aldo, I intend to use this in the "update game list" option and other options that may take a while to finish, that way the user will be aware that the command is still running and the notification will stop when the other command completes or show another one like "Done"

and in some important pop up information

**Comment by aldostools** [comment]
Check the update https://github.com/aldostools/webMAN-MOD/commit/7e9e43606eb1d68e896b24a57afd9eef2b23b675

**Comment by LuanTeles** [comment]
Thanks Aldo

Wow that's amazing, using /refresh.ps3 i would get the notification of the SCAN XML and after about 2 minutes that my 182 games are found.

When. I asked you the command i intended to use like this

/popup.ps3$Scaning games..\n the XMB will be automatically reloaded when the process is complete;/refresh.ps3;/popup.ps3*;/play.ps3/dev_hdd0/PROISO/RELOADXMB.iso

and later i would add the text to the rco for multi language.

But now it's also showing a "progress" popup, it's showing my persistent popup and also another popup saying "Scan for content: X games" and it keeps increasing the count as more games got scanned and it intercalates with my popup, was it intentional or is it a side effect that the persistent notification added? maybe this progress status are the one that would be passed to the fps counter plugin if enabled?

I think would be amazing now if the progress notification can be kept without using my persintent popup, maybe a /refresh.ps3$ command could be created for this?


Also /popup.ps3* to disable the notification, makes a tripple beep, can it be removed?

**Comment by aldostools** [comment]
The "progress" notification is a feature that has been there since last year (or maybe more).

You can activate the persistent system information holding SELECT+START for about 5 seconds to see the progress or press SELECT+START at any moment to see the progress.

Alternatively you can use /popup.ps3$ to turn on the persistent info.

**Comment by LuanTeles** [comment]
That's nice, but isn't a way to use just refresh.ps3 and show the "progress" without any other notification?

for e.g: /popup.ps3$S;/refresh.ps3;/popup.ps3*;/play.ps3/dev_hdd0/PROISO/RELOADXMB.iso 

command will display the system info then the "Scan for content: 5 games" then the system info again and "Scan for content: 22 games" and so on that's kind annoying

maybe a /refresh.ps3$ so it will display only the default "Refresh XML: Scan for Content" and keeps showing the "progress" until the operation is done

i really liked this feature, that's why i'm asking for it, it's useful for people with large game collection that the scan takes a while and we have no visual feedback if it is scanning or not, just when the progress is done

**Comment by aldostools** [comment]
No. Currently it isn't possible to do exactly what you're requesting.

It would require to make additional changes, more testing and release another build.

**Comment by LuanTeles** [comment]
NP Aldo,

i'll stick with the custom text to show the "progress" i really liked this feature

Thanks for the persistent notification!

**Comment by LuanTeles** [comment]
@aldostools 

how to use it with a rco msg?

**Comment by LuanTeles** [comment]
@aldostools 

how to use a persistent bottom notification? If it is not possible, no problem I'm satisfied with the current implementation


I was trying it to avoid the alternating popup

/popup.ps3?msg_webman_scanning_content_notification&rco=explore_plugin&snd=1;/popup.ps3$msg_webman_scanning_notification&rco=explore_plugin;/refresh.ps3;/popup.ps3*;/play.ps3/dev_hdd0/PROISO/RELOADXMB.iso

So the persistent one would be a "Scanning..." message as a botton nortification and the progress as a normal popup

**Comment by aldostools** [comment]
The botton notification is not in the "persistent" loop. So the option is currently unavailable.
